config.prod.json

```diff
@@ -185,5 +185,8 @@
     },
     "permalink_prefix": "https://tchap.gouv.fr",
     "custom_translations_url_comments": "To add translations, edit /src/i18n/strings/tchap_translations.json. It gets copied over to webapp/i18n for serving.",
-    "custom_translations_url": "/i18n/tchap_translations.json"
+    "custom_translations_url": "/i18n/tchap_translations.json",
+    "tchap": {
```

Comment:
I would call it `tchap_features`.
Or `features_tchap`, and place it next to `features` in the file.
What do you think?

Reply:
I've chosen a generic name because we might use it for other matters than feature flagging. A nice fourre-tout for tchap things.